entry/entry.go

```diff
@@ -96,15 +172,15 @@ func (c *CborEntry) ToEntry(provider identityprovider.Interface) (*Entry, error)
 }
 
 // ToCborEntry creates a CBOR serializable version of an entry
-func (e *Entry) ToCborEntry() *CborEntry {
+func (e *Entry) ToCborEntry() interface{} {
```

Comment:
Why didn't you keep the explicit return type?

Reply:
It is an opaque type that can be serialized, except if I miss something we need to return an `interface{}` type here otherwise we won't be comply to the interface definition

Reply:
Ha ok, that's good then.